### ISSUE TITLE
Bundles example doesn't work with typed-emitter

### ISSUE THREAD

**Issue by hongaar** [hongaar/bandersnatch#299]
Running the bundle example from the readme (also in the `examples/bundle` dir) yields this error:

```
pkg@5.3.1
Error! This experimental syntax requires enabling one of the following parser plugin(s): 'flow, typescript' (1:7)
bandersnatch/examples/bundle/node_modules/typed-emitter/index.d.ts`
```

This is caused by https://github.com/andywer/typed-emitter/pull/12 and has since been fixed with https://github.com/andywer/typed-emitter/pull/16 but it's still pending a NPM release. Once that's in, we should upgrade `typed-emitter` and that should resolve this issue.

Related to https://github.com/hongaar/bandersnatch/pull/295

Thanks @arbue for reporting this!

**Comment by arbue** [comment]
Hi @hongaar 

Just wanted to inform you, that there is a new release in the andywer/typed-emitter package which should resolve this issue:

https://github.com/andywer/typed-emitter/releases/tag/v1.4.0

**Comment by hongaar** [comment]
Fixed with #368
